examples/analytics-3p-remote-frame-helper.js

```diff
@@ -0,0 +1,26 @@
+
+class AmpAnalyticsInstance {
```

Comment:
Just name the class `AmpAnalytics` or something else without `Instance` in the name. As it is now, an instance of this class is an AmpAnalyticsInstance instance, which is confusing.